integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ClusterTemplateViewEntity.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.it.cloudbreak.newway.entity;
+
+import com.sequenceiq.cloudbreak.api.model.template.ClusterTemplateRequest;
+import com.sequenceiq.cloudbreak.api.model.template.ClusterTemplateViewResponse;
+import com.sequenceiq.it.cloudbreak.newway.AbstractCloudbreakEntity;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.Prototype;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+
+@Prototype
+public class ClusterTemplateViewEntity extends AbstractCloudbreakEntity<ClusterTemplateRequest, ClusterTemplateViewResponse, ClusterTemplateViewEntity> {
```

Comment:
I think it is mergeable with ClusterTemplateEntity.

Reply:
I deleted it and created a workaround for ClusterViewEntity in case of listing cluster templates